notebooks/text_classification/solutions/text_generation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p><strong>nit:</strong> Consider putting the comments in fully fledge sentences in markdown above rather than in the code. I personally find that too many comments within the code tend to make it harder to read, especially when it's obvious. For instance, since it's very clear that <code>print(f"Length of text: {len(text)} characters")</code>  does, I would not add a comment explaining we are printing the length of text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sounds good! Cleaned up comments throughout the whole lab. Thank you!</p>